src/Core/src/PerformanceTracker/Extensions/PerformanceMauiAppBuilderExtensions.cs

```diff
@@ -0,0 +1,31 @@
+using System.Diagnostics.Metrics;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Maui.Hosting;
+
+namespace Microsoft.Maui.Performance
+{
+    /// <summary>
+    /// Extension methods to register and configure performance monitoring services for a MAUI application.
+    /// </summary>
+    internal static class PerformanceMauiAppBuilderExtensions
+    {
+        /// <summary>
+        /// Adds .NET MAUI performance monitoring services to the application's dependency injection container.
+        /// </summary>
+        /// <param name="builder">The <see cref="MauiAppBuilder"/> to which performance monitoring is being added.</param>
+        /// <returns>The same <see cref="MauiAppBuilder"/> instance.</returns>
+        public static MauiAppBuilder ConfigurePerformance(
+            this MauiAppBuilder builder)
+        {
+            // Register the Meter
```

Comment:
Can this check `IsMetricsSupported` (`Guard()`) and return early?

Then it would never create a `Meter` at startup.

Reply:
Good point, right must not create a `Meter` at all if the feature is not enabled.

Reply:
Done.